src/utilities/util_main.c

```diff
@@ -9,8 +9,10 @@
 #include "util.h"
 
 const char *home = "."; /* Home directory */
-const char *progname;   /* Program name */
-                        /* Global arguments */
+/* Give users a hint in the help output if they're trying to read MongoDB data files. */
+const char *mongodb_config = "log=(enabled=true,path=journal,compressor=snappy)";
```

Comment:
There's a lot of stuff in the `wiredtiger_open` call but this is the stuff we care about.